bool IdleCmd
bool VoltageCmd
bool VelocityCmd
bool PositionCmd
